From 50b22c9ea9d159618b1a496169f0157b43133035 Mon Sep 17 00:00:00 2001
From: Jabiyeff <cebiyevanar@gmail.com>
Date: Sun, 3 Jul 2022 14:03:40 +0400
Subject: [PATCH] config: Add option for custom display-commonsys interfaces

Signed-off-by: Jabiyeff <cebiyevanar@gmail.com>
---
 config/BoardConfigQcom.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/config/BoardConfigQcom.mk b/config/BoardConfigQcom.mk
index 72933218..91e6729d 100644
--- a/config/BoardConfigQcom.mk
+++ b/config/BoardConfigQcom.mk
@@ -144,10 +144,12 @@ PRODUCT_SOONG_NAMESPACES += $(QCOM_SOONG_NAMESPACE)
 
 # Add display-commonsys to PRODUCT_SOONG_NAMESPACES for QSSI supported platforms
 ifneq ($(filter $(QSSI_SUPPORTED_PLATFORMS),$(TARGET_BOARD_PLATFORM)),)
+ifneq ($(TARGET_USES_CUSTOM_DISPLAY_INTERFACE),true)
 PRODUCT_SOONG_NAMESPACES += \
     vendor/qcom/opensource/commonsys/display \
     vendor/qcom/opensource/commonsys-intf/display \
     vendor/qcom/opensource/display
+endif # TARGET_USES_CUSTOM_DISPLAY_INTERFACE
 endif
 
 # Add data-ipa-cfg-mgr to PRODUCT_SOONG_NAMESPACES if needed
