From 30d05bcabc22b2448f82bf0353a4a726fb7cf160 Mon Sep 17 00:00:00 2001
From: Jabiyeff <cebiyevanar@gmail.com>
Date: Thu, 4 Aug 2022 19:05:33 +0000
Subject: [PATCH] config: Add option for custom display-commonsys interfaces

Signed-off-by: Jabiyeff <cebiyevanar@gmail.com>
---
 config/BoardConfigQcom.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/config/BoardConfigQcom.mk b/config/BoardConfigQcom.mk
index 70c947dbc..ab7c1c805 100644
--- a/config/BoardConfigQcom.mk
+++ b/config/BoardConfigQcom.mk
@@ -143,7 +143,9 @@ PRODUCT_SOONG_NAMESPACES += $(QCOM_SOONG_NAMESPACE)
 
 # Add display-commonsys-intf to PRODUCT_SOONG_NAMESPACES for QSSI supported platforms
 ifneq ($(filter $(QSSI_SUPPORTED_PLATFORMS),$(TARGET_BOARD_PLATFORM)),)
+ifneq ($(TARGET_USES_CUSTOM_DISPLAY_INTERFACE),true)
     PRODUCT_SOONG_NAMESPACES += vendor/qcom/opensource/commonsys-intf/display
+endif # TARGET_USES_CUSTOM_DISPLAY_INTERFACE
 endif
 
 # Add data-ipa-cfg-mgr to PRODUCT_SOONG_NAMESPACES if needed
